Lesson09_Task01_Alternative.py

```diff
@@ -0,0 +1,31 @@
+"""
+Lesson09 Task01.
+
+# write script that will read "input.txt" file
+and find there all characters from english alphabet
+# collect these characters and their positions in file
+# after info collected this info as text write
+to "output.txt" file in such format
+# ####################
+# <first found letter> -> pos1
+# <next_letter> -> pos2
+# <next_letter -> pos3
+# etc
+# #######################
+"""
+
+import re
+
+try:
+    with open('input.txt', 'r') as input_txt:
+        data = input_txt.read()
+        eng_letters = re.findall('[a-zA-Z]', data)
+        letter_pos = [(letter, data.index(letter)) for letter in eng_letters]
```

Comment:
This won't work for duplicate characters. But this is alternative solution not primary ))

Reply:
file with the incorrect solution was removed